### DISCUSSION TITLE
“静态文件CDN”设置

### DISCUSSION THREAD

**Discussion by Snitro** [Tokinx/Wing#91] [问答]
您好，请问“主题设置”中的“静态文件CDN”设置是否为配置网站中的图片、视频等资源的CDN呢，有没有简单的配置方法呢？十分感谢！

**Comment by Tokinx** [comment]
主题自带的这个功能很简单，没有很丰富的配置项，可以安装第三方插件实现自定义配置

**Comment by Snitro** [comment]
我看您的博客中图片文件都使用了cdn，是否是通过此配置完成的设置呢？

**Reply by Tokinx** [reply]
是的，是通过此配置完成设置的。
如果是七牛、又拍等CDN服务商，你可以使用一个域名加速你当前网站，然后在主题设置中的CDN填入加速后的地址。
例如：
1、通过cdn.a.com加速a.com（确保加速域名与源域名访问文件一致：cdn.a.com/test.txt = a.com/test.txt）
2、在主题设置中填入cdn.a.com
主题会将图片、附件等静态文件替换为加速后的域名，以达到加速静态文件访问的效果。
